src/Morphader.cpp

```diff
@@ -0,0 +1,324 @@
+#include "plugin.hpp"
+
+
+// equal sum crossfade, -1 <= p <= 1
+template <typename T>
+inline T equalSumCrossfade(T a, T b, const float p) {
+	return a * (0.5f * (1.f - p)) + b * (0.5f * (1.f + p));
+}
+
+// equal power crossfade, -1 <= p <= 1
+template <typename T>
+inline T equalPowerCrossfade(T a, T b, const float p) {
+	//return std::min(std::exp(4.f * p), 1.f) * b + std::min(std::exp(4.f * -p), 1.f) * a;
+	return std::min(exponentialBipolar80Pade_5_4(p + 1), 1.f) * b + std::min(exponentialBipolar80Pade_5_4(1 - p), 1.f) * a;
+}
+
+// TExponentialSlewLimiter doesn't appear to work as is required for this application.
```

Comment:
Hopefully this comment is clear but I can clarify if not. I will remove the commented code after we've discussed.